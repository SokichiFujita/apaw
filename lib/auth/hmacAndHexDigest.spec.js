const hmacAndHexDigest = require('./hashAndHexEncode');

describe('hmacAndHexDigest', () => {
  it('should create a hexidecimal string of input key and value pairs', () => {
    const hashedString = hmacAndHexDigest('key', 'value');
    expect(hashedString).toMatchSnapshot();
  });
});
