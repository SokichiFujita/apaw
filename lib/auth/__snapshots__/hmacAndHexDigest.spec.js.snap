// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hmacAndHexDigest should create a hexidecimal string of input key and value pairs 1`] = `"b58807fba080c6fe56737fb8504d00b0d026258340dd0b86cc5d6cfa927eb9e7"`;
